src/main/java/com/todoist/pojo/Colors.java

```diff
@@ -4,37 +4,60 @@
-    public static final int AQUA = 0xFF46D5F9;
-    public static final int RASPBERRY = 0xFFDA53AC;
-    public static final int CHERRY = 0xFFAA1D3F;
-    public static final int RUBY = 0xFFD0482E;
-    public static final int PISTACHIO = 0xFF83B822;
-    public static final int TEAL = 0xFF1DB3B1;
-    public static final int LAGOON = 0xFF128298;
-    public static final int SKY = 0xFF61B3FC;
-    public static final int SAPPHIRE = 0xFF1174C3;
-    public static final int ONYX = 0xFF000000;
-    public static final int STEEL = 0xFF777777;
-    public static final int FOREST = 0xFF149320;
-    public static final int OLIVE = 0xFFA39C1F;
-    public static final int TOMATO = 0xFFE53E1B;
-    public static final int MAGENTA = 0xFFE51AA3;
-    public static final int GRAPE = 0xFF9822E3;
-    public static final int PEACOCK = 0xFF2020E3;
-    public static final int SEA = 0xFF1483C3;
-    public static final int CHARCOAL = 0xFF555555;
-    public static final int CRIMSON = 0xFFAA1D3F;
-    public static final int EMERALD = 0xFF83B822;
-    public static final int NIGHT = 0xFF111111;
-}
+    public static final int BERRY_RED = 0xFFB8255F;
+    public static final int RED = 0xFFDB4035;
+    public static final int ORANGE = 0xFFFF9933;
+    public static final int YELLOW = 0xFFFAD000;
+    public static final int OLIVE_GREEN = 0xFFAFB83B;
+    public static final int LIME_GREEN = 0xFF7ECC49;
+    public static final int GREEN = 0xFF299438;
+    public static final int MINT_GREEN = 0xFF6ACCBC;
+    public static final int TEAL = 0xFF158FAD;
+    public static final int SKY_BLUE = 0xFF14AAF5;
+    public static final int LIGHT_BLUE = 0xFF96C3EB;
+    public static final int BLUE = 0xFF4073FF;
+    public static final int GRAPE = 0xFF884DFF;
+    public static final int VIOLET = 0xFFAF38EB;
+    public static final int LAVENDER = 0xFFEB96EB;
+    public static final int MAGENTA = 0xFFE05194;
+    public static final int SALMON = 0xFFFF8D85;
+    public static final int CHARCOAL = 0xFF808080;
+    public static final int GRAY = 0xFFB8B8B8;
+    public static final int TAUPE = 0xFFCCAC93;
+
+    public static final int DEFAULT_COLOR_INT = GRAY;
+    public static final int DEFAULT_COLOR_ID = 48;
+
+    private static final int[] COLORS = {
+            BERRY_RED,
+            RED,
+            ORANGE,
+            YELLOW,
+            OLIVE_GREEN,
+            LIME_GREEN,
+            GREEN,
+            MINT_GREEN,
+            TEAL,
+            SKY_BLUE,
+            LIGHT_BLUE,
+            BLUE,
+            GRAPE,
+            VIOLET,
+            LAVENDER,
+            MAGENTA,
+            SALMON,
+            CHARCOAL,
+            GRAY,
+            TAUPE
+    };
+
+    public static int getColor(int colorId) {
+        // Color IDs start at 30. To simplify, subtract that value and it can be mapped to the array's index.
+        colorId -= 30;
+
+        if (colorId >= 0 && colorId < COLORS.length) {
+            return COLORS[colorId];
```

Comment:
I think the ternary operator would be nice here (shorter and just one return statement). 😊